src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.Unix.cs

```diff
@@ -175,8 +175,7 @@ internal static SafeFileHandle OpenReadOnly(string fullPath, FileOptions options
         }
 
         internal static SafeFileHandle Open(string fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, long preallocationSize,
-                                            Interop.Sys.Permissions openPermissions = DefaultOpenPermissions,
-                                            Func<Interop.ErrorInfo, Interop.Sys.OpenFlags, string, Exception?>? createOpenException = null)
+                                            Interop.Sys.Permissions openPermissions = DefaultOpenPermissions)
```

Comment:
Isn't this actually a bug? Shouldn't `createOpenException` be passed through instead of it being dropped and `null` being passed in on line 182?

cc @adamsitnik @carlossanlop @Jozkee

Reply:
Undone the change